projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.scss

```diff
@@ -28,7 +28,7 @@
   width: 100%;
 
   &:focus-visible {
-    @include lg-inner-focus-outline();
+    @include lg-focus-outline;
```

Comment:
The focus for the accordion doesn't look exactly right. I feel like it should be internal to the box 🤔 worth checking with Alex:
![Screenshot 2023-04-25 at 09 42 29](https://user-images.githubusercontent.com/8397116/234223286-fd50797a-863f-4bf3-ba30-e7a827486c92.png)

The details component will be the same

Reply:
I've checked this with Alex and he said the focus outline should be kept outside of the box. So I've added a z-index for the focused element to fix this.

Reply:
Perfect thank you 👍 I will deploy in a sec so I can double check the changes 👍